src/design-system/text/text.css.ts

```diff
@@ -40,11 +40,17 @@ export const typography = recipe({
     color: {
       primary: sx({ color: '$text_primary' }),
       secondary: sx({ color: '$text_secondary' }),
+      grey: sx({ color: '$text_grey' }),
       accent: sx({ color: '$text_accent' }),
       highlight: sx({ color: '$data_blue' }),
       success: sx({ color: '$data_green' }),
       error: sx({ color: '$data_pink' }),
       warning: sx({ color: '$data_orange' }),
+      laceGradient: sx({
```

Comment:
Przemek will talk with Allan about that.

Reply:
@przemyslaw-wlodek @DominikGuzei I checked the code and this name `laceGradient` is commonly used in ui toolkit codebase. I think it's fine.